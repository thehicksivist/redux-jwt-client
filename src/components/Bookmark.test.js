import React from 'react'
import renderer from 'react-test-renderer'
import Bookmark from './Bookmark'

describe('Bookmark snapshot', () => {
    it('Should render as expected', () => {
        const tree = renderer.create(
            <Bookmark key='123' title='My Bookmark' url="http://www.mybookmark.com" _id="123" remove={() => { }} />
        )
        expect(tree.toJSON()).toMatchSnapshot()
    })
})