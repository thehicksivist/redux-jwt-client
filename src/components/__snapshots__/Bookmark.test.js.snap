// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bookmark snapshot Should render as expected 1`] = `
<li>
  My Bookmark
   (
  <a
    href="http://www.mybookmark.com"
  >
    Visit
  </a>
  )
  <button
    onClick={[Function]}
  >
    Delete!
  </button>
</li>
`;
